server/src/test/java/io/crate/action/sql/SessionTest.java

```diff
@@ -138,17 +137,16 @@ public void testGetParamType() {
             SessionContext.systemSessionContext());
 
         session.parse("S_1", "Select 1 + ? + ?;", Collections.emptyList());
-        assertThat(session.getParamType("S_1", 0), is(DataTypes.UNDEFINED));
-        assertThat(session.getParamType("S_1", 2), is(DataTypes.UNDEFINED));
+        session.ensureDescribed("S_1");
```

Comment:
isn't this testing that types are undefined without `describe`?
otherwise the followup assertions are somehow duplicates... maybe just remove this assertions without replacement?

Reply:
Pushed a fixup to split up the test cases.

I kept the second batch of `getParamType` assertions to ensure that the explicit `session.describe('S', "S_1");` call doesn't somehow mess with the results.